Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/m5spec.yml

```diff
@@ -0,0 +1,133 @@
+  components:
+  - type: Tag
+    tags:
+      - RMCRocketAmmo84mm
+  - type: CartridgeAmmo
+    proto: RMCProjectileRocket84mm
+    deleteOnSpawn: false
+    count: 1
+    spread: 0
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Guns/Ammunition/Explosives/m5spec.rsi
+    layers:
+    - state: m5_84mm_he
+      map: ["enum.AmmoVisualLayers.Base"]
+  - type: Item
+    size: Normal
+  - type: Appearance
+  - type: SpentAmmoVisuals
+    state: m5_84mm_he
+    suffix: false
+
+- type: entity
+  id: RMCProjectileRocket84mm
+  name: 84mm high-explosive rocket
+  noSpawn: true
+  description: Uh oh.
+  components:
+  - type: TriggerOnCollide
+  # TODO RMC14 Make TriggerOnCollide spawn the explosion on the OtherEntity tile more consistently, so the explosion is centered properly
+    fixtureID: projectile # no parent entity from BaseBulletTrigger because it has Clickable
+  - type: Physics
+    bodyType: Dynamic
+    fixedRotation: false
+  - type: Fixtures
+    fixtures:
+      projectile:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.15,-0.15,0.15,0.15"
+          # Bigger than bullets. Rectangle won't work since AABB shapes don't rotate. This should help significantly with explosion consistency.
+        hard: false
+        mask:
+        - Impassable
+        - BulletImpassable
+        restitution: 0.0
+        density: 20
+  - type: Projectile
+    impactEffect: BulletImpactEffect
+    damage:
+      types:
+        Blunt: 15
+    MaxFixedRange: 6 # 7 tiles
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Guns/Ammunition/Projectiles/m5spec_projectiles.rsi
+    layers:
+    - state: m5_84mm_he
+  - type: ExplodeOnTrigger
+  - type: Explosive
+    explosionType: RMC
+    maxIntensity: 30
+    intensitySlope: 10
+    totalIntensity: 290
+    maxTileBreak: 0
+  - type: CMExplosionEffect
+  - type: PointLight
+    radius: 3.5
+    color: orange
+    energy: 0.5
+
+# 84mm AP
+
+- type: entity
+  id: RMCRocket84mmAntiArmor
+  name: 84mm anti-armor rocket
+  parent: RMCRocket84mm
+  description: A rocket tube compatible with the M5-ATL loaded with an AP warhead. Capable of piercing and downing heavily armored targets. Deals very little to no splash damage. Inflicts guaranteed stun to most targets. Has high accuracy within 7 meters.
+  components:
+  - type: CartridgeAmmo
+    proto: RMCProjectileRocket84mmAntiArmor
+    deleteOnSpawn: false
+    count: 1
+    spread: 0
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Guns/Ammunition/Explosives/m5spec.rsi
+    layers:
+    - state: m5_84mm_ap
+      map: ["enum.AmmoVisualLayers.Base"]
+  - type: SpentAmmoVisuals
+    state: m5_84mm_ap
+    suffix: false
+
+- type: entity
+  id: RMCProjectileRocket84mmAntiArmor
+  name: 84mm anti-armor rocket
+  parent: RMCProjectileRocket84mm
+  noSpawn: true
+  components:
+  - type: Projectile
+    impactEffect: BulletImpactEffect
+    damage:
+      types:
+        CMArmorPiercing: 50
```

Comment:
Is this a leftover from old code? The damage type doesn't seem to exist (not that it should either)
```suggestion
```